ArtOfIllusion/src/artofillusion/RotateViewTool.java

```diff
@@ -414,6 +417,7 @@ private void moveChildren(ObjectInfo parent, Mat4 transform, UndoRecord undo)
   
   private void tilt(WidgetMouseEvent e, ViewerCanvas view, Point clickPoint)
   {
+	System.out.println("Tilting!!!!");
```

Comment:
You probably meant to remove this line.

Reply:
I probably did...

It was a leftover from the original view re-do

Reply:
fixed.